tooling/noir_js/src/backend/barretenberg.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable  @typescript-eslint/no-explicit-any */
```

Comment:
```suggestion
```

Reply:
I'm trying to remove the Ptr type from barretenberg.js since that explicitly uses the /dest/browser and seems to not work well when trying to compile to both esm and cjs so we may need to keep this a little longer